pages/common/stat.md

```diff
@@ -11,7 +11,7 @@
 
 `stat -t {{file}}`
 
-- Show filesystem information:
+- Show file system information:
```

Comment:
We use `filesystem` along the whole doc pages
```suggestion
- Show filesystem information:
```

Reply:
Ahh that was my bad.
A while back I changed `file system` to `filesystem` thinking `file system` was wrong.
When undoing it, I must've missed that one.